### ISSUE TITLE
Taesd for SD 1.5

### ISSUE THREAD

**Issue by andreineamtu** [madebyollin/taesd#17]
Do you have a version for Stable Diffusion v1.5 that can be loaded using from_single_file method as VAE?
I tried it and got this error:
"diffusers.loaders.single_file_utils.SingleFileComponentError: Failed to load AutoencoderKL. Weights for this component appear to be missing in the checkpoint."

**Comment by madebyollin** [comment]
No, TAESD is not an `AutoencoderKL` model so you can't load TAESD using `AutoencoderKL.from_single_file`.

To load TAESD in diffusers, you can instead use `AutoencoderTiny.from_pretrained`, like this:
```python

import torch
from diffusers import AutoencoderTiny
taesd = AutoencoderTiny.from_pretrained("madebyollin/taesd", torch_dtype=torch.float16)
```

For more detailed information, see the [AutoencoderTiny](https://huggingface.co/docs/diffusers/en/api/models/autoencoder_tiny) diffusers docs.

**Comment by andreineamtu** [comment]
Yes that's how I did, but this means to load it from HuggingFace. 
I wanted to load it from my local machine. Is that possible?

**Comment by madebyollin** [comment]
Yeah, `from_pretrained` supports [local folders](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/modeling_utils.py#L362), so you can download the `taesd` folder with `git clone https://huggingface.co/madebyollin/taesd` and then construct the model from that `taesd` folder using `AutoencoderTiny.from_pretrained("/path/to/taesd")`.
